src/components/ProfilePage/Avatar/Avatar.js

```diff
@@ -0,0 +1,10 @@
+import './avatar.css';
+import React from 'react';
+
+const Avatar = React.forwardRef((props, ref) => {
+    return (
+        <img className="img"  ref = {ref}onClick={props.onClick} src={props.src} width={props.width} alt="user's avatar" accept=".jpg, .jpeg, .png"></img>
```

Comment:
Fix code style (formatting):
`ref = {ref}onClick={props.onClick}`

Reply:
I think it is better to capitalize the sentence for a user:
`alt="user's avatar"`